### PR TITLE
Support for Ordered/Linked Items

### LINKED ISSUE #4: Implement linked items

### PR DESCRIPTION
**Pull request by tmilewski**
Closes #4

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ const node = require('when/node')
 const posthtml = require('posthtml')
 const loader = require('posthtml-loader')
 
+const isArray = (obj) => {
+  return Object.prototype.toString.call(obj) === '[object Array]'
+};
```
Comment (jescalan) [inline lib/index.js]: You can use `Array.isArray`

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "spike-contentful",
   "description": "Contentful CMS plugin for spike",
-  "version": "0.1.2",
+  "version": "0.2.0",
```
Comment (jescalan) [inline package.json]: don't boost the version in a PR, maintainer can do this after merge

File: test/index.js [hunk 3]
```diff
@@ -69,24 +69,6 @@ test('initializes with "limit" filter', (t) => {
...
-  // )
-  //
-  // t.throws(
-  //   () => {
-  //     opts.filters.limit = 101
-  //     console.log(opts)
-  //     new Contentful(opts)
-  //   }, // eslint-disable-line
-  //   'ValidationError: [spike-contentful constructor] option "limit" must contain less than or equal to 100 items'
-  // )
```
Comment (jescalan) [inline test/index.js]: 👍 👍 👍

File: package.json (hunk 2 above)
Comment (tmilewski) [inline package.json]: ... and, if I am the maintainer?

**Comment by jescalan** [comment]
👍

**Pull request merged**